.gitignore

```diff
@@ -259,7 +259,7 @@ paket-files/
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
 *.pyc
-Dependencies/Assembly-CSharp-firstpass.dll
-Dependencies/Assembly-CSharp.dll
-/Dependencies/Assembly-CSharp-firstpass_publicized.dll
-/Dependencies/Assembly-CSharp_publicized.dll
+Assembly-CSharp-firstpass.dll
+Assembly-CSharp-firstpass_publicized.dll
+Assembly-CSharp.dll
+Assembly-CSharp_publicized.dll
```

Comment:
Okay, what???
What is this supposed to achieve?
This will only ignore the assembly files _if they're found in the root directory_

Reply:
Ok I was wrong